src/browser/actions/makeBalancesActions.js

```diff
@@ -6,10 +6,10 @@ import generateDAppActionId from './generateDAppActionId';
 
 export const ID = 'balances';
 
-export default function makeBalancesActions(sessionId, requestId) {
+export default function makeBalancesActions(sessionId, requestId, call = getBalances) {
```

Comment:
Trying to use jest to stub this `getBalances` default export proved pretty challenging, and it felt like a good opportunity to just use dependency injection instead.  I decided to go that route instead, and I think we should do the same for all our `makeXxxActions` that are used for the nOS API.